person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = @@counter
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  public
+
+  def can_use_services?
+    @age >= 18 || @parent_permission
```

Comment:
- [ ] Great job working on this milestone. Since you have a private method `of_age?`, in the public method `can_use_services?`, you just have to call the method and use. It could be  `of_age?() || @parent_permission`